ArduCopter/AP_Arming.cpp

```diff
@@ -322,7 +322,7 @@ bool AP_Arming_Copter::motor_checks(bool display_failure)
         const uint32_t esc_mask = tcan->get_present_mask();
         uint8_t escs_missing = 0;
         uint8_t first_missing = 0;
-        for (uint8_t i = 0; i < 16; i++) {
+        for (uint8_t i = 0; i < 32; i++) {
```

Comment:
might as well change to the define? Not all boards get the 32.

Reply:
I would change it to NUM_SERVO_CHANNELS.
This NUM_SERVO_CHANNELS is the result of a 16 or 32 decision.